tests/deepsparse/evaluation/test_registry.py

```diff
@@ -1,51 +0,0 @@
-# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
You are moving those tests to sparsezoo right?

Reply:
Yes they are there!